stix2elevator/extension_definitions.py

```diff
@@ -246,6 +262,22 @@
         "version": "1.0.0",
         "extension_types": ["property-extension"]
     },
+    "report": {
+        "id": "extension-definition--faebed16-db10-4824-8538-62d9cd38b117",
```

Comment:
if you are saying both in the "same extension", why aren't they the exact same object for both grouping and report, especially since they add the same property - that makes sense.  I'm not sure if just because they have the same url for the schema that they can't be different.  

Also, these extension definitions are a bit of a hack for the elevator.

Reply:
Whoops, I had a typo in my comment: `id` not `if`